src/main/java/racingcar/domain/MotorRacing.java

```diff
@@ -33,9 +46,9 @@ public MotorRacingDisplayResults racing(final int raceRound){
     }
 
     private MotorRacingDisplayRoundResult allCarsStartRacing() {
-        final List<Integer> collect = racingCars.stream()
+        final List<RacingResult> collect = racingCars.stream()
                                           .peek(this::runMotorRace)
```

Comment:
peek 은 자주 사용되는 메소드가 아닌데 선택하신 특별한 이유가 있으실까요 ?

```
According to its JavaDocs, java.util.Stream.peek() “exists mainly to support debugging” purposes. Although this does not mean that using it for other purposes is discouraged, relying on peek() without careful consideration can lead to error-prone code such as:
If the stream pipeline does not include a terminal operation, no elements will be consumed and the peek() action will not be invoked at all.
As long as a stream implementation can reach the final step, it can freely optimize processing by only producing some elements or even none at all (e.g. relying on other collection methods for counting elements). Accordingly, the peek() action will be invoked for fewer elements or not at all.
This rule raises an issue for each use of peek() to be sure that it is challenged and validated by the team to be meant for production debugging/logging purposes.
```